src/css/themes/po-theme-default.css

```
@@ -687,11 +687,11 @@ po-button {
     DATEPICKER RANGE
   \*------------------------------------*/
 
-  --color-datepicker-range-background-color-disabled: var(--color-neutral-light-90);
+  --color-datepicker-range-background-color-disabled: var(--color-neutral-light-20);
```

```suggestion
  --color-datepicker-range-background-color-disabled: var(--color-neutral-light-80);
```
para manter a customização atual